cli/cli.js

```diff
@@ -1,17 +1,17 @@
 #!/usr/bin/env node
 'use strict'
 
+const fs= require('fs')
+const path = require('path')
 const importJsx = require('import-jsx')
 const { h, render } = require('ink')
 const meow = require('meow')
 const open = require('react-dev-utils/openBrowser')
-
-const dev = require('@compositor/kit-dev')
-
+const log = require('@compositor/log')
 const config = require('pkg-conf').sync('kit')
 
 const App = importJsx('./src/App')
-const pkg = require('./package.json')
+const parseArgs = require('./lib/parse-args')
```

Comment:
Split this out because it might get more complicated in the future, but can move it back into this module if that makes more sense

Reply:
I like it split out tbh 💯